.github/workflows/ci.yml

```
@@ -180,34 +180,33 @@
-            if echo "$RECENT_COMMITS $PR_TITLE" | grep -iE "(feat|feature|add|implement|new)" > /dev/null; then
-              VERSION_TYPE="minor"
-              echo "🚀 Detected FEATURE: Using minor version bump"
-            elif echo "$RECENT_COMMITS $PR_TITLE" | grep -iE "(break|breaking|major)" > /dev/null; then
-              VERSION_TYPE="major"
-              echo "💥 Detected BREAKING CHANGE: Using major version bump"
+            if [ -n "$EXPLICIT_VERSION" ]; then
+              # Explicit version found in PR title - use it directly
+              CLEAN_VERSION=$(echo "$EXPLICIT_VERSION" | sed 's/v//')
+              echo "📌 Explicit version specified in PR title: $EXPLICIT_VERSION"
+              echo "Setting version directly to: $CLEAN_VERSION"
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)